### PR TITLE
26 fixed positional embeddings

### PR DESCRIPTION
**Pull request by Alikerin**

### CODE REVIEW

File: instageo/model/run.py [hunk 1]
```diff
@@ -377,8 +386,10 @@ def compute_metrics(
             dict: A dictionary containing 'iou', 'overall_accuracy', and
```
Comment (ZeLynxy) [inline instageo/model/run.py:386]: Let's add roc_auc in the docstring also

Comment (ZeLynxy) [inline instageo/model/run.py:386]: Let's add roc_auc in the docstring also

File: instageo/model/dataloader.py [hunk 2]
```diff
@@ -22,17 +22,53 @@
...
 import torch
+import xarray as xr
 from absl import logging
 from PIL import Image
+from rasterio.crs import CRS
 from torchvision import transforms
 
-from instageo.data.hls_utils import open_mf_tiff_dataset
+
+def open_mf_tiff_dataset(
```
Comment (ZeLynxy) [inline instageo/model/dataloader.py:38]: is there a reason why this function has been moved here? I see that we don't need it completely as it is in the data module. Maybe we can have a lighter version (remove for instance mask and crs) and change the name ?

**Review by ZeLynxy — CHANGES_REQUESTED** [review]

File: instageo/model/dataloader.py (hunk 2 above)
Comment (Alikerin) [reply instageo/model/dataloader.py:38]: The reason why it was moved here is to ensure independence between data and model components. Before you have to supply the earthdata account credentials to use the model component which is not ideal. 

Your comment sparked a thought and I see we rather need to apply the mask properly because we want to get the same processing on the tile during `sliding_window_inference` as we do when creating chips
